docs/MO_DG/prepare_model/convert_model/Converting_Model.md

```diff
@@ -1,32 +1,52 @@
-# General Conversion Parameters {#openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model}
+# Setting Input Shapes {#openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model}
 
-To get the full list of general (framework-agnostic) conversion parameters available in Model Optimizer, run the following command:
+Paragraphs below provide details about specifing input shapes for model conversion.
+
+## When to Specify `--input_shape` Command Line Parameter <a name="when_to_specify_input_shapes"></a>
+There are situations when Model Optimizer is unable to deduce input shapes of the model. In this case, user has to specify input shapes explicitly
+using `--input_shape` parameter. For example, run the Model Optimizer for the TensorFlow* MobileNet model with the single input
+and specify input shape `[2,300,300,3]`.
 
 ```sh
-mo --help
+mo --input_model MobileNet.pb --input_shape [2,300,300,3]
 ```
 
-Paragraphs below provide useful details on relevant parameters.
+If a model has multiple inputs, `--input_shape` must be used in conjunction with `--input` parameter.
+The parameter `--input` contains a list of input names for which shapes in the same order are defined via `--input_shape`.
+For example, launch the Model Optimizer for the ONNX* OCR model with a pair of inputs `data` and `seq_len` 
+and specify shapes `[3,150,200,1]` and `[3]` for them.
+
+```sh
+mo --input_model ocr.onnx --input data,seq_len --input_shape [3,150,200,1],[3]
+```
 
-## When to Specify --input_shape Command Line Parameter <a name="when_to_specify_input_shapes"></a>
-There are situations when Model Optimizer is unable to deduce input shapes of the model, for example, in case of model cutting due to unsupported operations.
-The solution is to provide input shapes of a static rank explicitly.
+The parameter `--input_shape` allows to override original input shapes to ones compatible for a given model.
+Dynamic shapes, i.e. with dynamic dimensions, in the original model can be replaced with static shapes for the converted model, and vice versa.
```

Comment:
This is kinda unclear 
Dynamic shapes, i.e. with dynamic dimensions, in the original model can be replaced with static shapes for the converted model, and vice versa.
Does it mean - Dynamic shapes that have dynamic dimensions in the original model can be replaced with static shapes in the converted model, and vice versa.

Reply:
yes. Please clarify How to rephrase better.